### PR TITLE
my work for review on ticket 6

### PR DESCRIPTION
**Pull request by stephenL81**

### CODE REVIEW

File: app.test.js [hunk 1]
```diff
@@ -134,3 +135,48 @@ test('should respond with a 400 if the provided id is not valid (eg not a number
...
+        return request(app)
+        .get('/api/articles/999/comments')
+        .expect(404)
+        .then(({body})=>{
+    
+            expect(body.msg).toBe('Not Found')
+
+    })
+})
+})
```
Comment (robert112233) [inline app.test.js:191]: You are missing a 200 test for an existing article with no comments (article 2 I think?), you should expect to receive an empty array and a 200 status code 👍

File: model.js [hunk 2]
```diff
@@ -9,16 +9,28 @@ function returnTopics(){
     })
 }
 
-
+function returnArticleComments(articleId){
+    return db.query("SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC" ,[articleId])
+    .then(({rows})=>{
+        if(rows.length === 0){
```
Comment (robert112233) [inline model.js:15]: This logic will wrongly reject for articles with no comments, you should use a checkExists before this, to differentiate between existing articles with no comments (returns []) and non existing articles with no comments (also returns []). Drop me a Dm for any clarification on this :)

**Review by robert112233 — CHANGES_REQUESTED** [review]
Implement the missing test and a checkExists and submit another PR for review, excellent work though my friend!

**Comment by stephenL81** [comment]
will submit another pull request
